lib/tool-bar-view.coffee

```diff
@@ -23,13 +21,13 @@ module.exports = class ToolBarView extends View
     nextItem
 
   removeItem: (item) ->
-    index = @items.indexOf item
-    @items.splice index, 1
-    element = atom.views.getView item
```

Comment:
Con you elaborate why you removed `atom.views.getView` here, but kept it [here](https://github.com/suda/tool-bar/pull/101/files#diff-ee6e3704a4d781721ea4e95448f76196L18)?
It's to long ago, but I thought there was a reason why we used views...